rln/src/circuit.rs

```diff
@@ -248,28 +248,28 @@ fn json_to_g2(json: &Value, key: &str) -> Result<G2Affine> {
 }
 
 // Converts JSON to a VerifyingKey
-fn to_verifying_key(json: serde_json::Value) -> Result<VerifyingKey<Curve>> {
+pub fn to_verifying_key(json: &serde_json::Value) -> Result<VerifyingKey<Curve>> {
```

Comment:
I suggest to use ark serilize: 

```
pub fn to_verifying_key(json: &serde_json::Value) -> Result<VerifyingKey<Curve>> {
    let data: Vec<u8> = serde::de::Deserialize::deserialize(json)?;
    let vk = VerifyingKey::<Curve>::deserialize_uncompressed(data.as_slice()).unwrap();
    Ok(vk)
}
```

Benchmark result: 
<img width="903" alt="image" src="https://github.com/vacp2p/zerokit/assets/45395078/c5a95190-78b6-404e-aa96-b602e8620c13">

Reply:
right, I agree, however, this PR is just to add the benchmark *so* that it can be optimized. Will address it in a follow up PR.